### ISSUE TITLE
Save results in a specified directory

### ISSUE THREAD

**Issue by martemyev** [mfem/mfem#44]
Hi All,

I'm wondering if there is a way to specify an output directory where I want all the visualization files to be saved? Currently, if I save results in VisIt files, MFEM creates a bunch of directories and corresponding files in a directory from where I call a program. I didn't find any options to change that. That would be especially helpful when one works on a computer cluster where it's typical to have a small home directory where the compiled binary and input files are, and another much larger directory where the big output files are saved.

Thanks,
Mikhail

**Comment by mlstowell** [comment]
Hi, Mikhail,

This is something I've been wanting as well.  I'll be sure to bring it up at our next meeting.

Thanks,
Mark

**Comment by v-dobrev** [comment]
The name in the `VisItDataCollection` constructor can be an absolute path, e.g.
```
   VisItDataCollection visit_dc("/path/to/my/dir/results", pmesh);
```
One drawback is that the path will appear in the `.mfem_root` files and if you move the data you'll have to edit out the prefix `/path/to/my/dir/` from the `.mfem_root` files.

**Comment by v-dobrev** [comment]
Another option is to switch the directory (using e.g. [`chdir`](http://pubs.opengroup.org/onlinepubs/9699919799/functions/chdir.html)) just before calling `visit_dc.Save()` and then switch back if needed.

**Comment by martemyev** [comment]
Hi Mark and Veselin,
Thank you for the suggestions! I changed the code a little bit for these needs. I'll create a pull request for you to check out.

Thanks,
Mikhail

**Comment by tzanio** [comment]
Resolved in pull request #48.
